### PR TITLE
Add documentation for cloudupgrade step

### PR DESCRIPTION
**Pull request by witekest**

### CODE REVIEW

File: scripts/mkcloud [hunk 1]
```diff
@@ -1236,6 +1236,7 @@ Steps:
     addupdaterepo:  addupdate repos defined in UPDATEREPOS= (URLs separated by '+')
     runupdate:      run zypper up on the crowbar node
                     (compute nodes are automatically updated via chef run)
+    cloudupgrade:   prepare the admin node for upgrade
```
Comment (skazi0) [inline scripts/mkcloud]: this step actually performs whole upgrade not just the admin node preparation

File: scripts/mkcloud [hunk 2]
```diff
@@ -1236,6 +1236,7 @@ Steps:
     addupdaterepo:  addupdate repos defined in UPDATEREPOS= (URLs separated by '+')
     runupdate:      run zypper up on the crowbar node
                     (compute nodes are automatically updated via chef run)
+    cloudupgrade:   execute the admin node upgrade
```
Comment (skazi0) [inline scripts/mkcloud]: this is still not correct. this step performs whole system upgrade end-to-end.

File: scripts/mkcloud [hunk 3]
```diff
@@ -1236,6 +1236,7 @@ Steps:
     addupdaterepo:  addupdate repos defined in UPDATEREPOS= (URLs separated by '+')
     runupdate:      run zypper up on the crowbar node
                     (compute nodes are automatically updated via chef run)
+    cloudupgrade:   execute the cloud upgrade
```
Comment (skazi0) [inline scripts/mkcloud]: maybe it would be good to add some note about `upgrade_cloudsource` here

File: scripts/mkcloud [hunk 4]
```diff
@@ -1236,6 +1236,8 @@ Steps:
     addupdaterepo:  addupdate repos defined in UPDATEREPOS= (URLs separated by '+')
     runupdate:      run zypper up on the crowbar node
                     (compute nodes are automatically updated via chef run)
+    cloudupgrade:   upgrade the cloud to the version specified
+                    by the `upgrade_cloudsource` parameter
```
Comment (skazi0) [inline scripts/mkcloud]: this is a shell script. the backticks call the command inside.

Comment (witekest) [reply scripts/mkcloud]: Could you please suggest how the text should look like?

Comment (skazi0) [reply scripts/mkcloud]: just remove the backticks :o)

**Review by JanZerebecki — APPROVED** [review]

**Pull request merged**